src/lib.rs

```diff
@@ -77,6 +77,16 @@ impl Builder {
         self
     }
 
+    pub fn whitelisted_function<T: Into<String>>(mut self, arg: T) -> Builder {
```

Comment:
Yeah, they look the same as `whitelisted_type`, and break similarly:

```
    pub fn whitelisted_type<T: Borrow<str>>(mut self, arg: &T) -> Builder {
        self.options.whitelisted_types.insert(arg);
        self
    }
```

I'll fix up `_type` and push.